src/PowerShellEditorServices/Session/PowerShell3Operations.cs

```diff
@@ -32,27 +33,11 @@ public void PauseDebugger(Runspace runspace)
             out DebuggerResumeAction? debuggerResumeAction)
         {
             IEnumerable<TResult> executionResult = null;
-
-            using (var nestedPipeline = currentRunspace.CreateNestedPipeline())
+            using (var pwsh = PowerShell.Create())
```

Comment:
I'd use the type name rather than `var` here, since it's not apparent from `Create()`.

Reply:
I think static `Create` factory methods are common enough (especially in the PS engine) that they might as well be the same syntax as `new PowerShell()`.  I don't feel particularly strongly about it or anything, just weighing in.

Reply:
I left it as is this time since it's the pattern we currently use everywhere.